lib/rules/declaration-property-value-no-unknown/README.md

```diff
@@ -139,3 +139,145 @@ a { top: --foo(0); }
 ```css
 a { top: --foo(10px); }
 ```
+
+### `atRulesSyntax: { "at-rule": { "prelude": "syntax", "descriptors": { "descriptor": "syntax" } } }`
```

Comment:
I'm unsure if it's necessary to go into this much detail.
Validation is not strictly based on this comment.
https://github.com/stylelint/stylelint/pull/8124#discussion_r1841089623